ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -78,6 +79,26 @@
   // the count of renegotiation after initial handshake done
   public final Counter sslRenegotiationCount;
 
```

Comment:
We have an issue for 2: #334. If we are fixing it, please also take a look at whether all the other metrics are getting registered as well, and removing unused methods as described in that issue.